### PR TITLE
change product title bottom divider to full width

### LINKED ISSUE #3633: Make the bottom divider of the Product Title row full width
> Could we make the bottom divider of the Product Title row full width? That would be more consistent because we currently have 3 different sizes of dividers on this screen.

Screenshot_20210225-130527
<sup>Tested with WCAndroid 6.1-rc-1 on Pixel 3 Android 11 on comparative-moth.jurassic.ninja (WP 5.7-RC1 JP 9.4 WC 5.1.0-rc.1) logged in as testuser01121.</sup>

<sup>(internal reference: p5T066-1Ov-p2#comment-7141 hat tip @Garance91540)</sup>

### PR DESCRIPTION
**Pull request by rossanafmenezes**
Fixes #3633

## Changes

Set the start margin of the Product Title bottom divider to 0 (it was set to 16dp). 

## Screnshots

| Before | After |
| ---- | ---- |
|  |  |
|  |  |

I realize it's really hard to see the difference with the light theme because of the screenshot border.

## Testing

- Navigate to the Products Screen
- tap on an existing product
- Note the Product name bottom divider is not set to full width. 

Update release notes:

- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/res/layout/product_property_editable_view_layout.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <View
         android:id="@+id/divider"
         style="@style/Woo.Divider"
-        android:layout_marginStart="@dimen/major_100"
+        android:layout_marginStart="@dimen/minor_00"
```
Comment (rossanafmenezes) [inline WooCommerce/src/main/res/layout/product_property_editable_view_layout.xml:34]: I also noticed that removing the margin altogether would have the same effect on the app. Dear reviewer, please feel free to suggest otherwise, if you think removing it would be a better option.

Comment (malinajirka) [reply WooCommerce/src/main/res/layout/product_property_editable_view_layout.xml:34]: Thanks for pointing this out @rossanafmenezes, I always appreciate when the author gives a headsup to the reviewer about possible discussion points ;)!

The only difference I'm aware of is that if we specify `android:layout_marginStart` in `@style/Woo.Divider`, the explicit `android:layout_marginStart="@dimen/minor_00"` used here will override the margin specified in `@style/Woo.Divider`. This might be a benefit or a downside - difficult to tell :) (my gut feeling is that it doesn't make any difference since the chances that we'll set margin in `@style/Woo.Divider` are close to zero).

In cases like this, I usually check if it's used somewhere else in the codebase to make sure I don't introduce inconsistencies. There are ten files where `android:layout_marginStart="@dimen/minor_00"` is used. This would suggest that in general we omit the marginStart attribute when it's `0` but at the same time this PR isn't introducing inconsistencies.

To summarize, I think the approach you took is fine ;).

**Review by malinajirka — APPROVED** [review]
Thanks @rossanafmenezes! The change looks good to me, great job ;)

P.S. I've left a no-action-needed comment in case you'd find my reasoning useful.
